pkg/identity/metricsd.go

```diff
@@ -72,7 +73,14 @@ func Metricsd(idConfig *config.IdentityConfig, stopChan <-chan struct{}) (error,
 
 	if idConfig.TargetDomainRoles != "" && idConfig.RoleCertDir != "" {
 		for _, domainrole := range strings.Split(idConfig.TargetDomainRoles, ",") {
-			exporter.Files = append(exporter.Files, strings.TrimSuffix(idConfig.RoleCertDir, "/")+"/"+domainrole+".cert.pem")
+			targetDomain, targetRole, err := athenz.SplitRoleName(domainrole)
+			if err != nil {
+				continue
```

Comment:
```suggestion
				log.Warnf("Failed to initialize metrics exporter with TARGET_DOMAIN_ROLES: %s, err: %s", idConfig.TargetDomainRoles, err.Error())
				continue
```

Reply:
@WindzCUHK 
Let me not include "failed to" as it simply skips with `continue` for individual malicious element of target_domain_role

Reply:
maybe "fail to monitor ... role", plx help to fix the wordings